### PR TITLE
data: TanmayBhosale

### PR DESCRIPTION
**Pull request by TanmayBhosale**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5410"></a>

### CODE REVIEW

File: data/TanmayBhosale.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Tanmay Bhosale",
+  "displayStatsPublic" : true,
```
Comment (Ananya2001-an) [inline data/TanmayBhosale.json:3]: We don't need this prop anymore due to latest updates.
```suggestion

```

**Review by Ananya2001-an — CHANGES_REQUESTED** [review]
Just some minor changes. Rest looks good!

**Comment by TanmayBhosale** [comment]
Made the changes! Thank you for the input!

**Review by Ananya2001-an — APPROVED** [review]
Looks good! Thanks for contributing 😊

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**